dds_web/api/files.py

```diff
@@ -220,49 +220,123 @@ def get(self):
         files_folders = list()
 
         # Check project not empty
-        with DBConnector(project=project) as dbconn:
-            # Get number of files in project and return if empty
-            num_files = project.num_files
-            if num_files == 0:
-                return {
-                    "num_items": num_files,
-                    "message": f"The project {project.public_id} is empty.",
+        if project.num_files == 0:
+            raise EmptyProjectException(project=project)
```

Comment:
I don't think listing files on an empty project should raise BAD_REQUEST. I think of it as doing `ls` on an empty directory. I think the previous behaviour returning info that is empty is better.

Reply:
Agree. In particular, since Researchers can list on projects with the _In Progress_ status.

Reply:
Should Researchers be able to list project contents of projects with `In Progress` status? It means the project is not ready to be accessed by the Researcher after all.

Reply:
My gut feeling was _No_ and this is what I also was about to point out during the review of your PR [when Ina already merged it](https://github.com/ScilifelabDataCentre/dds_web/pull/833#issuecomment-1032651676). But I acknowledge that it isn't a big deal.

Reply:
True. Added a check.

Reply:
@MatthiasZepper It's in this PR now

Reply:
@inaod568: Thank you very much! Great!